### PR TITLE
Add MAE and L2 loss to docs

### LINKED ISSUE #12751: L1 Loss Documentation Issue
## 📚 Documentation

The documentation for torch.nn.L1Loss needs to say that it's the same as MAE loss super clearly in the documentation. Everyone knows what MAE is, and not nearly as many people know what L1 is. I've had a bunch of students come to my office hours with the documentation page open saying "MAE isn't in PyTorch, what's going on here?" That shouldn't be happening. Also, you kind of mix notations by calling MAE L1 and calling MSE MSE, not L2.

### PR DESCRIPTION
**Pull request by Kaixhin**
Fixes #12751

### CODE REVIEW

**Review by ssnl — APPROVED** [review]
Thanks @Kaixhin !

**Comment by justinkterry** [comment]
Was this not merged? What's going on here?

**Comment by SsnL** [comment]
@justinkterry it is merged in https://github.com/pytorch/pytorch/commit/84ce3ab47e4da0ca4727c9daf125138a7642935a
